app/components/market/submit/submit-market-application.component.ts

```diff
@@ -61,6 +61,19 @@ export class SubmitMarketApplicationComponent implements OnInit {
         this.modeState = mode;
     }
 
+    public getToolTip(): string {
+        if (this.isFormValid()) {
+            return "Click to submit form";
+        }
+        return "Form is not valid";
+    }
+
+    public isFormValid() {
+        return (this.modeState ===  Modes.NewPoolAndJob && this.jobParams.valid && this.poolParams.valid) ||
```

Comment:
you should just be able to have this.formValid

Reply:
Different mode will have different conditions for submit